noxfile.py

```diff
@@ -289,15 +289,15 @@ def linkcheck(session: nox.sessions.Session):
     )
 
 
-@nox.session(python=PY_VER[-1], venv_backend="conda")
+@nox.session(python=PY_VER, venv_backend="conda")
 @nox.parametrize(
     ["ci_mode"],
     [True, False],
     ids=["ci compare", "full"],
```

Comment:
N.B. I *was* going to ask for the options here to be clarified a bit, in terms of what you see in the "nox -l" lisiting.
But from offline conversation with @trexfeathers, there is a subsequent improvement to that coming soon in another PR.

Reply:
#4583 🙂